src/components/editor/EditorAppController.jsx

```diff
@@ -34,6 +34,8 @@ if (process.env.NODE_ENV === 'production') {
 export default class EditorAppController extends BaseComponent {
   constructor(props) {
     super(props);
+    this.restartServer = this.restartServer.bind(this);
+    this.pingServer = this.pingServer.bind(this);
```

Comment:
You can delete this line, the only reason for using the `.bind(this)` is if you need to call `this` as in `this.safeSetState` or `this.state` etc. and `this.pingServer` doesn't call `this` as far as I can see.